src/monattack.cpp

```diff
@@ -165,7 +182,7 @@ void mattack::acid(monster *z, int index)
 
     int t;
     int junk = 0;
-    if( !z->sees( t, target ) ||
+    if( !z->sees( target, t ) ||
```

Comment:
I'm sure there's a good reason for these but the code as written seemed to work.  What's up?